homeassistant/components/device_tracker/locative.py

```diff
@@ -76,21 +76,24 @@ def _handle(self, data):
         device = data['device'].replace('-', '')
         location_name = data['id'].lower()
         direction = data['trigger']
+        
+        kwargs = {
+           'dev_id': device,
+           'location_name': location_name,
+           'gps': (data['latitude'], data['longitude'])
```

Comment:
Can you please import some of these constants from `homeassistant.consts`? Like `ATTR_LATITUDE` and `ATTR_LONGITUDE`